db/migrate/20231101182340_create_post.rb

```diff
@@ -0,0 +1,13 @@
+class CreatePost < ActiveRecord::Migration[7.1]
+  def change
+    create_table :posts do |t|
+      t.string :title
+      t.text :text
+      t.integer :comments_counter
+      t.integer :likes_counter
+      t.references :user, null: false, foreign_key: true
```

Comment:
- [ ] 🚀 Impressive work so far in your implementation! 👏 However, I'd like to bring up a minor detail in your code that can further enhance it and ensure it aligns perfectly with all the project requirements.

   - 👉 In your `post` table, I noticed that you're referencing the `user` column on the `user` table as a `foreign_key`. According to the project's Entity-Relationship Diagram (ERD) and the explicit requirements, the `posts` table should be linked to the `users` table via the `author_id` column, not the `user` column.

**📜 Here's a quick reminder from the project requirements:**
   <img width="997" alt="Screenshot 2023-11-01 at 19 33 38" src="https://github.com/SaveryIV/blog-app-on-Rails/assets/107192076/db30f6b3-8a0e-4ad3-8e07-f3f842679cb9">
   
 Happy coding! 😊🚀👨‍💻